.env.example

```diff
@@ -1,7 +1,7 @@
 APP_NAME="Laravel Admin Panel"
 APP_ENV=local
 APP_KEY=base64:O+TrollIDwasHEREtMG9kBc+/Q32exQLusNVhnq558w=
-APP_DEBUG=true
+APP_DEBUG=false
```

Comment:
I will be reverting this change, as this project is starting point for other's project, and they will need to have this enabled. They can turn this setting off, if they want. But, by default, debugbar will be turned on.